VirtualMachineSetup.md

```diff
@@ -0,0 +1,152 @@
+# Virtual Machine Setup
+
+Follow the steps below to configure a Virtual Machine for development at VotingWorks. This guide assumes you are using Mac OS, if you are not modify the instructions as appropriate. 
```

Comment:
lmk if you want me to write up how to do this on linux, in case someone doesn't want to use a mac.

Reply:
Cool, I think its fine for now but if someone mentions wanting to do it on linux then it would be great to have you help with that. I don't think we need to do it until there is a clear need though.